src/nvim/msgpack_rpc/channel.c

```diff
@@ -619,8 +621,7 @@ static void unsubscribe(Channel *channel, char *event)
   free(event_string);
 }
 
-/// Close the channel streams/job. The channel resources will be freed by
-/// free_channel later.
+/// Close the channel streams/job and free the channel memory.
```

Comment:
Since you changed the comment: "resources" is more accurate than "memory" because it also closes the libuv handle.